content/newsletter/026.md

```diff
@@ -0,0 +1,203 @@
+efficient data access, and interoperability. The vision is of global astronomy
+archives connected via the VO to form a multiwavelength digital sky that can be
+searched, visualized, and analyzed in new and innovative ways. VO projects
+worldwide working toward this vision are already providing science capabilities
+with new tools and services. This newsletter, aimed at astronomers, highlights
+VO tools and technologies for doing astronomy research, recent papers, and
+upcoming events.
+
+---
+
+# IVOA NEWS
+
+{{< side-image image="/images/ivoa_logoc2.jpg" position="right" floating="true" alt="IVOA Logo" >}}
+
+
+## IVOA Interoperability meeting – Malta, November 2024
+
+Simon O'Toole
+
+The Northern Fall Interoperability meeting was held on 15-17 November in Valletta, Malta, after the ADASS meeting. As with all IVOA meetings since the COVID pandemic, it was a hybrid meeting, with a large number of attendees in person (113 in person, 49 online). Sessions were recorded and posted online. There were many great discussions and some big decisions were made, outlined below.
+
+{{</ side-image >}}
+
+
+<h3 style="text-align: left;">High Energy Interest Group</h3>
+
+A new Interest Group focussed on High Energy Astrophysics was formally approved at the Malta Interop. This had followed over a year of planning and many discussions and a session at the May 2024 Interop in Sydney. The chair of the Interest Group is Bruno Khelifi from CNRS, and the vice chair is Janet Evans from CfA.
+
+The HEIG held a joint plenary session with the Time Domain Interest Group where we heard about multi-messenger astronomy and the transient nature of high energy astrophysics. This session led to a renewed focus on the VOEvent standard, and a splinter session organised to discuss the path moving forward.
+
+<h3 style="text-align: left;">Protocol Transitioning Tiger Team</h3>
+
+The P3T held its final session in Malta – a lot of great work by the team to move IVOA standards into the OpenAPI framework. Initial work around UWS was presented by Josh Fraustro and further discussions around more complex standards took place. The team presented a set of potential next steps, and the transition to OpenAPI will now be the responsibility of the working groups.
+
+<h3 style="text-align: left;">Highlights</h3>
+
+There were plenty of other highlights from this Interop, including but definitely not limited to the following:
+* Further discussions on Digital Object Identifiers and other persistent unique IDs.
+* A lot of work on Parquet format in the VO, including:
+  * HATS or the Hierarchical Adaptive Tiling Scheme, which is a Parquet-based partitioning format for very large all-sky catalogues. This is especially important in the LSST-era and beyond.
+  * How to bring VOTable and Parquet together in a sensible way
+* Uploads and user-managed tables in TAP services
+* Improvements to how Single Sign On is handled in the VO.
+
+<br>
```

Comment:
The `<br>` does not seem necessary. Can it be removed? (Same for following ones.)

Reply:
> I added these `<br>`s  when reviewing with the make preview. If I don't add a `<br>` after the list, then no line space is added, between the list and the following paragraph, which doesn't look good. And I added the `<br>` before the sentence 'Members of the  astronomical...' to make this slightly separate from the above article. 

So blank lines don't work in these cases?